dataland-api-key-manager/src/main/kotlin/org/dataland/datalandapikeymanager/DatalandApiKeyManager.kt

```diff
@@ -1,15 +1,20 @@
 package org.dataland.datalandapikeymanager
 
 import org.dataland.datalandapikeymanager.configurations.OpenAPIConfiguration
```

Comment:
What happens if i request ownership of a company more than 1 time?
Different cases; 1. I request it multiple times and no claim was resolved. 2, i have ownership of a company, but request it nevertheless again. 3. i have claimed ownership, but it was denied and i claim it again

Reply:
Since we do not want to maintain a database table in this script we are only able to address 2.) which I did in the endpoint.

Reply:
ok